Description: Compatibility with Ruby 1.9
  Patch from RedHat Bugzilla 831501, see
  https://bugzilla.redhat.com/show_bug.cgi?id=831501#c9 .
  Refreshed to match the current version in Debian.
Forwarded: https://bugs.launchpad.net/screenruler/+bug/925835
Author: Russell Harrison, Christian Hofstaedtler
Last-Update: 2013-03-20

Index: screenruler-0.960+bzr41/ruler_window.rb
===================================================================
--- screenruler-0.960+bzr41.orig/ruler_window.rb	2014-01-16 11:06:58.602608138 +0100
+++ screenruler-0.960+bzr41/ruler_window.rb	2014-01-16 11:06:58.594608187 +0100
@@ -16,7 +16,8 @@
  #  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  ###############################################################################
 
-require 'glade_window', 'ruler_popup_menu', 'canvas', 'unique_timeout'
+require 'glade_window', 'canvas', 'unique_timeout'
+require_relative 'ruler_popup_menu'
 
 Unit = Struct.new('Unit', :name, :tick_pattern, :units_per_pattern_repetition, :per_inch)
 
Index: screenruler-0.960+bzr41/screenruler.rb
===================================================================
--- screenruler-0.960+bzr41.orig/screenruler.rb	2014-01-16 11:06:58.602608138 +0100
+++ screenruler-0.960+bzr41/screenruler.rb	2014-01-16 11:06:58.598608162 +0100
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-#coding: utf-8
+# -*- encoding: utf-8; -*-
 
  ###############################################################################
  #  Copyright 2011 Ian McIntosh <ian@openanswers.org>
@@ -48,7 +48,10 @@ SETTINGS_FILE_NAME = 'settings.yml'
 puts _('Loading libraries...')
 
 require 'addons_ruby'									# for multi-file 'require'
-require 'gtk2', 'settings', 'addons_gtk', 'ruler_window', 'preferences_window', 'help_window'
+require 'gtk2', 'settings', 'addons_gtk'
+require_relative 'ruler_window'
+require_relative 'preferences_window'
+require_relative 'help_window'
 
 ###################################################################
 # Main
